plugin-hrm-form/src/components/caseList/filters/DateRangeFilter.tsx

```diff
@@ -194,10 +197,42 @@ const DateRangeFilter: React.FC<Props> = ({
 
   const handleClear = () => {
     updateWorkingCopy(undefined);
+    resetDateValidation();
   };
 
   const dividerStyle = { border: 'none', height: '1px', backgroundColor: 'rgb(216, 216, 216)' };
 
+  const resetDateValidation = () => {
+    setDateValidations(prev => ({ ...prev, to: false, from: false, error: '', errorColor: '' }));
+  };
+
+  const handleDateValidation = () => {
+    const today = new Date();
+
+    if (currentWorkingCopy.to < currentWorkingCopy.from) {
+      setDateValidations(prev => ({
+        ...prev,
+        to: true,
+        from: false,
+        error: `The end date selected cannot be before the start date`,
+        errorColor: '1px solid red',
+      }));
+      return;
+    }
+
+    if (name !== 'followUpDateFilter' && (currentWorkingCopy?.from > today || currentWorkingCopy?.to > today)) {
+      setDateValidations(prev => ({
+        ...prev,
+        // eslint-disable-next-line no-unneeded-ternary
+        to: currentWorkingCopy?.to > today ? true : false,
```

Comment:
Why do we need to disable the lint warning instead of just changing the line to

```suggestion
        to: currentWorkingCopy?.to > today,
```
?

Reply:
I never thought of this :)